pkg/cdi/container-edits.go

```diff
@@ -138,6 +175,27 @@ func (e *ContainerEdits) Validate() error {
 	return nil
 }
 
+// Append other edits into this one. If called with a nil receiver,
+// allocates and returns newly allocated edits.
+func (e *ContainerEdits) Append(o *ContainerEdits) *ContainerEdits {
+	if o == nil || o.ContainerEdits == nil {
+		return e
```

Comment:
This return does not allocate a new edit if the receiver is `nil`. Is that a concern?

Reply:
No, it's deliberate choice. Appending nothing to a nil edit returns/keeps the edit nil, instead of converting it to an empty edit.